doc/xroad_catalog_installation_guide.md

```diff
@@ -111,12 +101,32 @@ Add the X-Road repository’s signing key to the list of trusted keys:
 sudo rpm --import https://artifactory.niis.org/api/gpg/key/public
 ```
 
-Install the `xroad-confclient` module:
+## 2.4 Installation
+
+The installable software consists of `xroad-catalog-collector` and `xroad-catalog-lister` modules. Both are provided
+as RPM packages.
+
+```bash
+sudo yum install xroad-catalog-lister xroad-catalog-collector
+```
+
+Or alternatively:
 
 ```bash
-sudo yum install xroad-confclient
+rpm -i install xroad-catalog-lister xroad-catalog-collector
 ```
```

Comment:
Why do we provide an alternative method here? It might be a bit confusing since we don't for the other cases, e.g. when installing `yum-utils` a bit earlier in this document. I would maybe leave this out of the document or then explain when the alternative should be used if it is required in certain cases.

Reply:
I removed the alternative way.